architecture/core.rst

```diff
@@ -800,42 +800,52 @@ Shared Memory
 Shared Memory is a block of memory that is shared between the non-secure and the
 secure world. It is used to transfer data between both worlds.
 
-Shared Memory Allocation
+Shared Memory References
 ========================
-The shared memory is allocated by the Linux driver from a pool ``struct
-shm_pool``, the pool contains:
 
-    - The physical address of the start of the pool
-    - The size of the pool
-    - Whether or not the memory is cached
-    - List of chunk of memory allocated.
+The shared memory is allocated and managed by the non-secure world, i.e the
+Linux OP-TEE driver. Secure world only considers the individual shared buffers,
+not their pool. Each shared memory is referenced with associated attributes:
+    - Buffer start address and byte size
+    - Cacheability of the buffer
```

Comment:
https://english.stackexchange.com/questions/251159/is-cacheability-a-word-technical-word

Reply:
you expect a change in current proposal?
Maybe change to "Cache attributes of the shared memory buffer"?